pkg/scorecard_result.go

```diff
@@ -146,6 +147,11 @@ func FormatResults(
 func (r *ScorecardResult) AsString(showDetails bool, logLevel log.Level,
 	checkDocs checks.Doc, writer io.Writer,
 ) error {
+	if writer == os.Stdout {
+		fmt.Fprintln(writer)
+	}
+	fmt.Fprintln(writer, "RESULTS\n-------")
```

Comment:
hmm, not sure about these yet.

Reply:
See if this explanation helps: This was done to print "RESULTS" header to the output file before printing Aggregated Score and the scores table. Otherwise, it will print only the Aggregated Score and the scores table.

Reply:
i'm still in favor of keeping these in their original spot